### ISSUE TITLE
Page Loading UX

### ISSUE THREAD

**Issue by viankakrisna** [Automattic/Picard#47]
Can we implement something like http://chieffancypants.github.io/angular-loading-bar/ with Picard? I came from angular and quite new with react. I saw that there are no direct feedback whenever user click on links other than the url changing. 

I think that it is important to give user something to see that we are actually loading something in the background, maybe something like http://cezary.github.io/react-loading/. But i'm still figuring out on how to implement it in Picard (or use that component in react).

**Comment by jacklenox** [comment]
Yes, this is absolutely a good idea. However, right now it's not a focus for me – as and when I get time I want to get all of the basics of a theme working within Picard. In the long run, I think it will be better if Picard is a starter theme with no purely aesthetic styling (like Underscores). However, this would still be a nice thing to have, in the same way that Underscores has basic support for a responsive nav.
